mkosi/resources/mkosi.md

```diff
@@ -488,30 +488,21 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
     metadata is always synced and and packages can be downloaded during
     the build.
 
-`PackageManagerTrees=`, `--package-manager-tree=`
+`SandboxTrees=`, `--sandbox-tree=`
 :   Takes a comma separated list of colon separated path pairs. The first
-    path of each pair refers to a directory to copy into the OS tree
-    before invoking the package manager. This option is similar to the
-    `SkeletonTrees=` option, but installs the files to a subdirectory of
-    the workspace directory instead of the OS tree. This subdirectory of
-    the workspace is used to configure the package manager. If the
-    `mkosi.pkgmngr/` directory is found in the local directory it is used
-    for this purpose with the root directory as target (also see the **Files**
-    section below). If not configured in any way this value will default to
-    the same value of `SkeletonTrees=`.
+    path of each pair refers to a directory to copy into the mkosi
+    sandbox before executing a tool. If the `mkosi.sandbox/` directory
+    is found in the local directory it is used for this purpose with the
+    root directory as target (also see the **Files** section below). If
+    not configured in any way this value will default to the same value
+    as `SkeletonTrees=`.
 
     `mkosi` will look for the package manager configuration and related
-    files in the configured package manager trees. Unless specified
-    otherwise, it will use the configuration files from their canonical
-    locations in `/usr` or `/etc` in the package manager trees. For
-    example, it will look for `etc/dnf/dnf.conf` in the package manager
-    trees if `dnf` is used to install packages.
-
-    `SkeletonTrees=` and `PackageManagerTrees=` fulfill similar roles. Use
-    `SkeletonTrees=` if you want the files to be present in the final image. Use
-    `PackageManagerTrees=` if you don't want the files to be present in the final
-    image, e.g. when building an initrd or if you want to refer to paths outside
-    of the image in your repository configuration.
```

Comment:
With the rename and the removal of this paragraph we should consider scrapping the defaulting to the skeleton tree.